### DISCUSSION TITLE
How to use "Whisper Advanced" tab in SE 3.6.13 beta

### DISCUSSION THREAD

**Discussion by MiskolcMiskolc** [SubtitleEdit/subtitleedit#7172] [Q&A]
Hello everyone! I've been using SE a lot recently and that speech to text function is just amazing, however I wonder if its possible to limit the max character length using that command line tab in the beta version. What command should I use and how because I'm not the best programmer and I didn't really find any discussions here or on reddit. Thanks for the help in advance!
![image](https://github.com/SubtitleEdit/subtitleedit/assets/140797920/239235eb-765d-497f-b005-dd7c3b4295fc)

**Comment by Cyprien-22** [comment]
I found the character limit in the command line to not give me good results. 

You can find , options -> Settings -> General there are options for length of captions per line and more. 

There is also a Break/split long lines tool that's under both Tools and Auto Translate drop down buttons. 

The second option I found easier to work with, but gave me more timing adjustments. 

I haven't yet tried the general options to limit character length yet. It may give better timing results, but I had really strange results with the command line options. 

Hope the above helps in the meantime.
